src/main/kotlin/ch/wisv/choice/document/service/DocumentServiceImpl.kt

```diff
@@ -36,9 +36,8 @@ class DocumentServiceImpl(val documentRepository: DocumentRepository,
     }
 
     override fun storeDocument(document: Document): Document {
-        if (document.file != null) {
-            fileRepository.saveAndFlush(document.file)
-        }
+        if(document.file != null)
+            fileRepository.saveAndFlush(document.file!!)
```

Comment:
`!!` is niet nodig als je net een null check hebt gedaan lijkt me?

Reply:
Ja klopt, maar kotlin merkt dit niet op tijdens compilation